frontend/ui/src/pages/Channels/Providers/Airy/ChatPlugin/sections/EditChatPlugin.tsx

```diff
@@ -33,8 +33,8 @@ export const EditChatPlugin = ({channel, host, updateConnection}: EditChatPlugin
   };
 
   const ConnectContent = () => {
-    const [displayName, setDisplayName] = useState(channel.metadata?.name || '');
-    const [imageUrl, setImageUrl] = useState(channel.metadata?.imageUrl || '');
+    const [displayName, setDisplayName] = useState((channel && channel.metadata?.name) || '');
```

Comment:
nitpick: Since we are already using the Elvis operator here, may aswell write `channel?.metadata?.name`, right?

Reply:
true!! will fix